docs/manual.rst

```diff
@@ -2358,13 +2359,13 @@ constructor to create an R-tree that you can query with another geometric object
 
   `New in version 1.4.0`.
 
-The `query` method on `STRtree` returns a list of all geometries in the tree that
-intersect the provided geometry argument. If you want to match geometries of a
-more specific spatial relationship (eg. crosses, contains, overlaps), consider
-performing the query on the R-tree, followed by a manual search through the
-returned subset using the desired binary predicate.
+The `query` method on `STRtree` returns a list of all geometries in the tree whose
```

Comment:
@kannes what would you think about "which have extents that intersect" ?

Reply:
I like it!